edu-blog-vue/components/AuthorBio.vue

```diff
@@ -39,29 +39,29 @@ aside {
   border-bottom: 1px solid lightgray;
 }
 
-.bio {
+.author-bio {
   display: grid;
   padding-top: 2rem;
   grid-template-rows: 50% 50%;
 }
 
 @media screen and (min-width: 481px) {
-  .bio {
+  .author-bio {
     grid-template-columns: auto auto;
   }
 }
 
 @media screen and (min-width: 1200px) {
-  .bio {
+  .author-bio {
     grid-template-columns: 20% 80%;
   }
 }
 
-.bio p {
+.author-author-bio p {
```

Comment:
I guess that's a typo. `.author-bio p` or as in the other tags you add an additional class.